webpack.config.prod.js

```diff
@@ -1,23 +1,35 @@
 const path = require('path');
 const HWP = require('html-webpack-plugin');
-module.exports = {
+module.exports = {  
     mode: 'production',
     entry: path.join(__dirname, '/src/index.js'),
     output: {
        filename: 'build.js',
-       path: path.join(__dirname, '/prod/dist')},
+       path: path.join(__dirname, '/dev/dist')},
     module:{
        rules:[{
-          test: /\.js$/,
+          test: /\.(js|jsx)$/,
           exclude: /node_modules/,
           loader: 'babel-loader',
           options: {
             presets: ['@babel/preset-env',
                       '@babel/react',{
                       'plugins': ['@babel/plugin-proposal-class-properties']}]
         }
+       },
+       {
+         test: /\.css$/,
+         use: ['style-loader', 'css-loader']
+       },
+       {
+         test: /\.(jpe?g|png|gif|svg)$/i, 
+         loader: 'url-loader'
```

Comment:
your builded app can not find images. Please, look again to the documentation how to copy images to the build folder: how to copy images to the dist directory - https://webpack.js.org/guides/asset-management/#loading-images

Reply:
I didn't get this point. Please help me with what I'm doing wrong here in our next Q&A session.
When I"m running my application it is loading images at my side.

Reply:
My comment was about situation when I run ```npm run build:dev``` command, go to the dist folder and open **index.html** file. In this case the app cannot find images, but I expect that dist folder after build has all files that I need for the correct app work